errai-jaxrs/errai-jaxrs-client/src/test/java/org/jboss/errai/enterprise/jaxrs/client/shared/QueryParamTestService.java

```diff
@@ -59,7 +59,11 @@
   @GET
   @Path("/5")
   public Set<String> getWithQueryParamSetOfStrings(@QueryParam("id") Set<String> id);
-  
+
+  @GET
+  @Path("/5t")
+  public Set<String> getWithQueryParamSetOfStringsTreeSet(@QueryParam("id") Set<String> id);
+
```

Comment:
why is this here in alignment PR?

Reply:
brought in by the PR @tiagobento raised to fix failing tests in errai

Reply:
ok